network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html

```diff
@@ -22,16 +22,11 @@
       {% endfor %}
       </ul>
     {% else %}
-    {# Show the tag's own "main" tag (i.e. first tag in its tag list) as each blog card's visible tag #}
-      {% with tag=page.specific.tags.first %}
-        {% if tag %}
-          {% if root %}
-            {# If we have a "root" context variable, we know this card is generated on an index page (or index page subroute) #}
-            <a class="main-tag d-block mt-3" href="{% routablepageurl root.specific "entries_by_tag" tag.slug %}">{{ tag }}</a>
-          {% else %}
-            {# If we do not have a "root" context variable, this card is being generated outside of an index page, and so needs to use its parent as root #}
-            <a class="main-tag d-block mt-3" href="{% routablepageurl page.get_parent.specific "entries_by_tag" tag.slug %}">{{ tag }}</a>
-          {% endif %}
+      {% with category=page.specific.category.first %}
```

Comment:
this should purely be a refactor PR, let's not slip in code _changes_ =)

Reply:
I didn't that's [the problem](https://github.com/mozilla/foundation.mozilla.org/pull/3649#issuecomment-531397383) 😭 Those changes are from a separate branch so, I'm not sure how this even happened.